source-code/sdk-js/src/adapter-sveltekit/vite-plugin/transforms/refactoring.md

```diff
@@ -209,27 +427,181 @@ JavaScript files can run on the server and on the client. So we need to detect t
       // result => 'import { get } from "svelte/store";const fn = () => { const { i } = getRuntimeFromContext();get(i)("test") }'
       ```
 
+Possible implementation:
+
+```js
+function transformJs(ast, config) {
+  const aliases = replaceSdkImports(ast)
+	aliases.forEach(({exportN, aliasN}) =>
+    // Wrap all occurences of `i`, `language`, etc with `get()`
+    if(!config.languageInUrl) identifiers(ast, aliasN).wrap(parse("get"))
```

Comment:
We only need to wrap code that is outside of functions we have wrapped before e.g. `load` etc.